docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md

```diff
@@ -1,30 +1,43 @@
+
+The tool that you use depends on whether the add-in is running in Edge or Internet Explorer. This is determined by the version of Windows 10 and the version of Office that are installed on the computer. For more information about determining which browser is being used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). 
 
-The F12 developer tools included in Windows 10 help you debug, test, and speed up your webpages. You can also use them to develop and debug Office Add-ins, if you are not using an IDE like Visual Studio, or if you need to investigate a problem while running your add-in outside the IDE. This article describes how to use the Debugger tool from the F12 developer tools in Windows 10 to test your Office Add-in.
 
 > [!NOTE]
 > The instructions in this article cannot be used to debug an Outlook add-in that uses Execute Functions. To debug an Outlook add-in that uses Execute Functions, we recommend that you attach to Visual Studio in script mode or to some other script debugger.
 
-## Prerequisites
+## When the add-in is running in Edge
 
-You need the following software:
+When the add-in is running in you can use the [Microsoft Edge DevTools](https://www.microsoft.com/p/microsoft-edge-devtools-preview/9mzbfrmz0mnj?activetab=pivot%3Aoverviewtab). 
 
-- The F12 developer tools, which are included in Windows 10. 
-    
-- The Office client application that hosts your add-in. 
-    
-- Your add-in. 
+1. Run the add-in. 
+
+2. Run the Microsoft Edge DevTools.
+
+3. In the tools, open the **Local** tab. Your add-in will be listed by its name.
+
+4. Click the add-in name to open it in the tools.
+
+5. Open the **Debugger** tab. 
+
+6. Choose the folder icon above the **script** (left) pane. From the list of available files shown in the dropdown list, select the JavaScript file that you want to debug.
+
+7. To set a breakpoint, select the line. You will see a red dot to the left of the line and a corresponding line in the **Call stack** (bottom right) pane.
+
+8. Run functions in the add-in as needed to trigger the breakpoint.
 
-## Using the Debugger
+## When the add-in is running in Internet Explorer
 
-You can use the Debugger from the F12 developer tools in Windows 10 to test add-ins from AppSource or add-ins that you have added from other locations. You can start the F12 developer tools after your add-in is running. The F12 tools display in a separate window and do not use Visual Studio.
+When the add-in is running in Internet Explorer, you can use the debugger from the F12 developer tools in Windows 10 to test your add-in (and also add-ins from AppSource or add-ins that you have added from other locations). You can start the F12 developer tools after the add-in is running. The F12 tools display in a separate window and do not use Visual Studio.
```

Comment:
Consider "The F12 tools display" -> "The F12 tools are displayed". I read the word display as a noun on the first pass. Maybe that's just a "me" problem...